astropy/tests/plugins/display.py

```diff
@@ -96,6 +96,8 @@ def pytest_report_header(config):
 
     # Helpers version
     try:
+        astropy_helpers_version = TESTED_VERSIONS['astropy_helpers']
+    except KeyError:
         from ...version import astropy_helpers_version
     except ImportError:
```

Comment:
Do we need to indent this?

Reply:
I don't understand

Reply:
The second `except` is to catch L101, which is indented, right? I rarely use multiple `except` so I can't tell whether not indenting L102 will work or not at a glance.

Reply:
Or maybe I am just confusing myself. Need to run but I'll look at this more carefully when I get the chance. 😬

Reply:
of course you're right, I misunderstood the first question of you suggesting we don't need the indent at all and I couldn't understand why you would say that.